### PR TITLE
Added version and commit reference CVE-2019-10216

### PR DESCRIPTION
**Pull request by tausif-rh**

### CODE REVIEW

**Comment by cve-team** [comment]
Please update the version affected to match the version in the updated description. Thank you.

File: 2019/10xxx/CVE-2019-10216.json [hunk 1]
```diff
@@ -48,14 +48,19 @@
...
+                "refsource": "CONFIRM"
             }
         ]
     },
     "description": {
         "description_data": [
             {
                 "lang": "eng",
-                "value": "It was found that the .buildfont1 procedure did not properly secure its privileged calls, enabling scripts to bypass `-dSAFER` restrictions. An attacker could abuse this flaw by creating a specially crafted PostScript file that could escalate privileges and access files outside of restricted areas."
+                "value": "It was found that in ghostscript up to and including version 9.27, the .buildfont1 procedure did not properly secure its privileged calls, enabling scripts to bypass `-dSAFER` restrictions. An attacker could abuse this flaw by creating a specially crafted PostScript file that could escalate privileges and access files outside of restricted areas."
```
Comment (iamleot) [inline 2019/10xxx/CVE-2019-10216.json]: This seems fixed in ghostscript 9.50.

Can you please change the sentence from:

````
 It was found that in ghostscript up to and including version 9.27, [...]
````

to:

````
 It was found that in ghostscript before version 9.50, [...]
````

Comment (iamleot) [reply 2019/10xxx/CVE-2019-10216.json]: ...and then add:

````
"version_affected": "<",
````

to: `"version_data"`?

**Comment by tausif-rh** [comment]
@iamleot Leonardo, added suggested Changes, please review.

**Pull request merged**